src/components/Restoration/BusinessName.vue

```diff
@@ -120,15 +114,19 @@ export default class BusinessName extends Vue {
 
   /** The request action code for this filing type. */
   get requestActionCode (): NrRequestActionCodes {
-    if (this.isRestorationFiling) return NrRequestActionCodes.RESTORE
+    if (this.isRestorationFiling) {
+      return NrRequestActionCodes.RESTORE
+    }
+
+    // fallback case - not used for now
     return null
   }
 
   /** Whether a new business legal name was entered. */
   get isNewName (): boolean {
     // Approved Name is null when we start
     // and is set when a name option is selected
-    return !!this.getNameRequestApprovedName // *** TODO: verify intial vs updated
+    return !!this.getNameRequestApprovedName
```

Comment:
Isn't `!!this.getNameRequestApprovedName` the same as `this.getNameRequestApprovedName`?